docs/colors.stories.mdx

```diff
@@ -33,15 +33,29 @@ Using the name of a palette color ensures if we update the theme your components
 | #582C83 | amethyst       | amethystTint       | amethystShade       | ❌           |
 | #8246AF | plum           | plumTint           | plumShade           | ❌           |
 | #003349 | slate          | slateTint          | slateShade          | ❌           |
-| #000000 | ❌            | ❌                 | ❌                  | blackOpacity |
-| #ffffff | ❌            | ❌                 | ❌                  | whiteOpacity |
+| #000000 | ❌             | ❌                 | ❌                  | blackOpacity |
+| #ffffff | ❌             | ❌                 | ❌                  | whiteOpacity |
+
 Tint, Shade and Opacity properties must be followed by a number.
 
 Where there is a collision between CSS strings and the palette color (e.g. gold) the palette color takes presedence.
+
 ```jsx
 <Typography color='brilliantGreen'>...</Typography>
 <Typography color='brilliantGreenTint20'>...</Typography>
 <Typography color='brilliantGreenShade30'>...</Typography>
 <Typography color='blackOpacity55'>...</Typography>
 <Typography color='whiteOpacity22'>...</Typography>
-```
\ No newline at end of file
+```
+
+## Design tokens
+
+Using design tokens ensures visual consistency across the application. If the designers update colors using a design tool, we have got always updated values.
+
+The name of the chosen design token must be always preceded by `--`, for example, if we want to use `colorsYang100` token in our code, we have to pass `--colorsYang100` as a prop. See [Design Tokens](https://zeroheight.com/2ccf2b601/p/217e24-design-tokens/b/46fb17) documentation for more information.
```

Comment:
This link leads to a doc page that is protected by a password. Not sure if it's necessary here tbh